Remove unnecessary highlight

MichalLankof opened the pull request:
Remove unnecessary highlight from "Example: asynchronous functions". The highlighted part is the same as in "Example: synchronous functions".

MichalLankof commented:
@googlebot I signed it!

kwalrath commented:
Nice catch! I'll see if I can fix the build.

kwalrath commented:
Ugh, fixing this problem will require fixing the following regex, which is what added the highlighting:

```
<?code-excerpt "async_await/bin/get_order.dart" replace="/(\s+\/\/ )(Imagine.*? is )(.*)/$1$2$1$3/g; /async|await/[!$&!]/g; /(Future<\w+\W)( [^g])/[!$1!] $2/g; /4/2/g"?>
```

I think we'll have to either explicitly ignore `Future<String> fetch` or explicitly look for the other two occurrences of `Future<String>`.

You don't happen to be really good at regular expressions, do you?

MichalLankof commented:
@kwalrath I am not quite sure how this works. Is this script just reading the template file from 'async_await/bin/get_order.dart` and replaces the contents using the provided regular expression? If so, why is there the output stored in the markdown file as well? 

If we need to change the regular expression it should be 
```
/(Future<\w+\W)( [^f])/[!$1!]$2/g
```
I have changed 2 things:

1. `[^g]` to `[^f]` to filter out part starting with an 'f' (fetchUserOrder)
2. removed space before `$2` because the first space is already captured in the second group and this only produced double spaces

This new regexp will fail over the already produced part because it contains double spaces. 
I will commit the new regexp and fix the double spaces, but let me know if I understood this correctly.

kwalrath commented:
That's it! The function name must have started with g (probably "get...") before, and the regex didn't get updated when the function name did.

Thank you for fixing this!
